src/content/app/genome-browser/components/browser-sidebar-toolstrip/BrowserSidebarToolstrip.tsx

```diff
@@ -133,6 +119,16 @@ const BrowserSidebarToolstrip = () => {
         onClick={() => toggleModalView(BrowserSidebarModalView.DOWNLOADS)}
         image={DownloadIcon}
       />
+      <ImageButton
+        className={classNames(
+          [layoutStyles.sidebarIcon],
+          [layoutStyles.navigateIcon]
```

Comment:
why so many arrays?

Reply:
sorry my bad, i was trying something else before and forgot to remove the square bracket. Done now.